handlers/namespace.go

```diff
@@ -0,0 +1,12 @@
+package handlers
+
+import (
+	"net/http"
+)
+
+// Swarm does not use namespaces, so we return an empty list. see https://github.com/openfaas-incubator/connector-sdk/pull/46
+func NamespaceLister() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("[]"))
```

Comment:
Can you set the content type too? 

Might be better to serialise an empty struct?

Reply:
👍